lab-steven/test/gallery-controller-test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const angular = require('angular');
+require('angular-mocks');
+
+describe('gallery controller', function(){
+  beforeEach(() => {
+    angular.mock.module('routesApp');
+    angular.mock.inject(($rootScope, $controller) => {
+      this.$rootScope = $rootScope;
+      this.galleryCtrl = new $controller('GalleryController');
+      // this.galleryCtrl.$onInit();
```

Comment:
can still keep the onInit call in there, as long as your controllers have their innards wrapped in onInit, too. this is your setup beforehand that is supposed to mock functionality of your galleryCtrl instance.